ProjectFiles/Assets/Scripts/Game/ServerGameLoop.cs

```diff
@@ -9,17 +12,47 @@ public class ServerGameLoop : IGameLoop
 
         public bool Init(string[] args)
         {
-
+            // TODO: make the config path an argument so we can swap configs later?
+            var config = LoadConfigOrDefault("server-config.json");
+            
             // Create world
             world = new World();
 
             // Start server
-            server = new Server(world);
+            server = new Server(world, config);
             var success = server.Start();
             
             return success;
         }
 
+        private ServerConfig LoadConfigOrDefault(string path)
```

Comment:
Should this be a static method in the `ServerConfig` class? I feel like this should be abstracted from the game loop.

Reply:
I think I agree but maybe Liam had a reason for this

Reply:
Yeah I agree, I'll make some changes now :)

Reply:
Okay cool :)